support-frontend/app/models/identity/requests/CreateGuestAccountRequestBody.scala

```diff
@@ -22,7 +22,10 @@ object CreateGuestAccountRequestBody {
 
   def fromEmail(email: String): CreateGuestAccountRequestBody = CreateGuestAccountRequestBody(email, PublicFields(Some(guestDisplayName(email))))
 
-  implicit val writesCreateGuestAccountRequestBody: Writes[CreateGuestAccountRequestBody] = Json.writes[CreateGuestAccountRequestBody]
+  implicit val writesCreateGuestAccountRequestBody: Writes[CreateGuestAccountRequestBody] = {
+    import com.gu.identity.model.play.WritesInstances.publicFieldsWrites
```

Comment:
identity-model-play has been added to provide type classes to serialise / deserialise case classes in identity-model using Play JSON